.pre-commit-config.yaml

```diff
@@ -2,18 +2,19 @@ minimum_pre_commit_version: "1.21"
 files: ^.*\.py$
 repos:
   - repo: https://github.com/PyCQA/bandit
-    rev: 1.7.4
+    rev: 1.7.5
     hooks:
     - id: bandit
       args: ["-c", "bandit.yaml"]
   - repo: https://github.com/trailofbits/pip-audit
-    rev: v2.4.15
+    rev: v2.5.2
     hooks:
       - id: pip-audit
   - repo: https://github.com/timothycrosley/isort
     rev: 5.12.0
     hooks:
     - id: isort
+      args: ["--profile", "black"]
```

Comment:
Ahahahahahhaa LOL

Reply:
I think this is not needed anymore, I'll remove it